src/Sarif.Driver/Sdk/ExportRulesDocumentationCommandBase.cs

```diff
@@ -46,13 +48,20 @@ internal void BuildRule(Skimmer<TContext> rule, StringBuilder sb)
         {
             sb.AppendLine($"## Rule `{rule.Moniker}`{Environment.NewLine}");
             sb.AppendLine($"### Description{Environment.NewLine}");
-            sb.AppendLine($"{rule.FullDescription.Text}{Environment.NewLine}");
+            sb.AppendLine($"{rule.FullDescription.Text ?? rule.ShortDescription.Text ?? "No description available."}{Environment.NewLine}");
             sb.AppendLine($"### Messages{Environment.NewLine}");
 
             foreach (KeyValuePair<string, MultiformatMessageString> message in rule.MessageStrings)
             {
-                sb.AppendLine($"#### `{message.Key.Split('_').Last()}`: {rule.DefaultLevel}{Environment.NewLine}");
-                sb.AppendLine($"{message.Value.Text}{Environment.NewLine}");
+                string ruleName = message.Key;
+                Match match = s_friendlyNameRegex.Match(message.Key);
+                if (match.Success)
+                {
+                    ruleName = match.Groups[2].Value;
```

Comment:
>2 [](start = 44, length = 1)

For code readability and also for robustness (because if you change the regex, this index might change): `match.Groups["friendlyName"].Value`